src/containers/pages/FocusInvestigation/active/index.tsx

```diff
@@ -2,8 +2,12 @@
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```

Comment:
Let's add some tests to this component that codify the calls made to Superset when the user filter is turned on.